tags/TagMapContext.md

```diff
@@ -0,0 +1,144 @@
+# Summary
+A `Tag` is a key-value pair that can be used to label anything that is associated
+with a specific operation, such as an HTTP request. These `Tag`s are used to
+aggregate measurements in a [`View`](https://github.com/census-instrumentation/opencensus-specs/blob/master/stats/DataAggregation.md#view) 
+according to unique value of the `Tag`s. The `Tag`s can also be used to filter (include/exclude)
+measurements in a `View`.
+
+# TagMap 
+`TagMap` is an abstract data type that represents collection of tags. 
+i.e., each key is associated with exactly one value, but multiple keys can be
+associated with the same value.  `TagMap` is serializable, and it represents
+all of the information that could be propagated across process boundaries.
+
+## TagKey
+
+A string or string wrapper, that contains two parts, scope and key. The scope
+of a tag is used to determine if a tag should be propagated or not.
+
+
+**TagKey Syntax:**  
+Scope and key are separated by ':'.
+```
+<Scope>:<Key>
+```
+
+**Scope**
+Scope of a `TagKey` defines a boundary within which the `TagKey` is relevant. For example 
+a `TagKey` http_route on frontend service could have a scope that spans across multiple services
```

Comment:
this is interesting. http route is not something I thought of as propagating that far. maybe something like endpoint (edge request)?

Reply:
What would be a typical value of tagkey edge-request?